src/sentry/projectoptions/defaults.py

```diff
@@ -135,6 +135,11 @@
     default=DEFAULT_PROJECT_PERFORMANCE_GENERAL_SETTINGS,
 )
 
+register(
```

Comment:
why is the default `1` here?

Reply:
i saw other uses it being `1` to mean true, but yeah i def. think we can make it a boolean so its less confusing!